app/modifiers/yasgui.js

```diff
@@ -0,0 +1,27 @@
+import { modifier } from 'ember-modifier';
+import config from 'frontend-contact-hub/config/environment';
+
+const defaultQuery =
+  config.yasgui.defaultQuery !== '{{YASGUI_DEFAULT_QUERY}}'
+    ? config.yasgui.defaultQuery
+    : `
+        SELECT * WHERE {
+            ?s ?p ?o .
+        } LIMIT 10
+    `;
+
+export default modifier(function yasgui(element /*, params, hash*/) {
+  Promise.all([
+    import('@triply/yasgui'),
+    import('@triply/yasgui/build/yasgui.min.css'),
```

Comment:
I also tried lazy-loading the css and that seems to work. Saves another 9KB for users who don't visit that page :smile: